t/002-content.t

```diff
@@ -1,5 +1,7 @@
 # vim:set ft= ts=4 sw=4 et fdm=marker:
 
+use strict;
+use warnings FATAL => 'all';
```

Comment:
Here is the duplicate code, because the following `use Test::Nginx::Socket::Lua` will introduces the same restriction

Reply:
Sorry, it's CLion auto completed, I will remove it